### ISSUE TITLE
Veteran and priority iconografy

### ISSUE THREAD

**Issue by gobyrne** [GCTC-NTGC/design-gc-digital-talent#70]
There are concerns around using letters to replace iconography for the tags "Veteran" as well as "Priority" which are used to determine the order in which users appear on a table.

We used letters as a starting point but this could prove challenging when we switch from English to French.

![Image](https://github.com/GCTC-NTGC/design-gc-digital-talent/assets/16977254/daccbd29-d920-4ab7-9b31-9f367198a904)

## To-do

Should we define a new icon with a different short form? e.g. "vet" ?

Should we create custom iconography? e.g. poppy flower, arrows?

**Comment by BrookeCameron** [comment]
Is this issue still relevant @gobyrne

**Comment by Jerryescandon** [comment]
Nope, the icons used here are going away with the screening and assessment tab

**Comment by BrookeCameron** [comment]
aren't you still using them in the table design?

**Comment by BrookeCameron** [comment]
![Image](https://github.com/user-attachments/assets/82985474-0c9c-4550-8ba2-a7719c08b724)

![Image](https://github.com/user-attachments/assets/fbedf744-e66e-4d39-bc1b-9e80d5513acb)

Putting these here incase they become useful @Jerryescandon

**Comment by Jerryescandon** [comment]
Table uses the full names 

<img width="939" height="576" alt="Image" src="https://github.com/user-attachments/assets/17c63c8e-44ed-409c-93f5-1dd8989a43a8" />
